server/nodejs/api/swagger.yaml

```diff
@@ -5045,22 +5045,27 @@ definitions:
         type: "string"
         description: "Alias to pay from"
       amounts:
-        type: "string"
-        description: "A stringified JSON object with aliases and amounts. { \"address\"\
-          :amount, ... } (numeric or string) The syscoin alias is the key, the numeric\
-          \ amount (can be string) in SYS is the value."
+        $ref: "#/definitions/AliasPayRequest_amounts"
```

Comment:
Actually can we change this object to AliasPayRequestAmounts?

Reply:
it is auto generated from the `./gen_server_yaml.sh`

Reply:
it is as you said in swagger.yaml in the root folder.